.github/workflows/release.yml

```diff
@@ -1,7 +1,8 @@
 name: Release
 
 on:
-  create
+  release:
+    types: [published]
```

Comment:
Will this still work for pre-releases (dunno what published means here)?

Reply:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release it is unclear, but I would *hope* so. We're about to find out.